app/domain/DolphinManager.js

```diff
@@ -174,6 +174,21 @@ export default class DolphinManager extends EventEmitter {
     }
   }
 
+  async playFileFromFrame(filePath, startingFrame) {
+    const uniqueId = crypto.randomBytes(3 * 4).toString('hex');
+
+    const jsonString = JSON.stringify({
+      mode: this.settings.mode || "normal",
+      replay: filePath,
+      startFrame: startingFrame,
+      isRealTimeMode: this.settings.isRealTimeMode || false,
+      commandId: uniqueId, // Indicates to Dolphin to play new replay
+    });
+
+    await this.writeCommFile(jsonString);
+    await this.runDolphin(true);
+  }
+
```

Comment:
Can you just have the `startFrame` be an optional parameter in the `playFile()` method? I don't think it makes sense to essentially write the same function twice.

Reply:
great idea, I should do that